homeassistant/components/sensor/fedex.py

```diff
@@ -33,11 +33,14 @@
 STATUS_DELIVERED = 'delivered'
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
-    vol.Required(CONF_USERNAME): cv.string,
-    vol.Required(CONF_PASSWORD): cv.string,
-    vol.Optional(CONF_NAME): cv.string,
+    vol.Required(CONF_USERNAME):
```

Comment:
Please limit the changes to this file to the fixes that you're submitting.

Reply:
Those are linting adjustments via a PEP8-compliant linter; no functionality changes exist in those lines. Should I reverse?

Reply:
Yes.